pandas/core/dtypes/inference.py

```diff
@@ -417,8 +417,7 @@ def is_dataclass(item):
 
     """
     try:
-        from dataclasses import is_dataclass
-
-        return is_dataclass(item) and not isinstance(item, type)
+        import dataclasses as dcs
+        return dcs.is_dataclass(item) and not isinstance(item, type)
```

Comment:
can we just `import dataclasses` and then return
```
dataclasses.is_dataclass(item) and not isinstance(item, type)
```

Reply:
Sure. This is perfect. I am working on this